src/modules/governance/components/actionComposer/actionComposerUtils.ts

```diff
@@ -368,7 +368,7 @@ class ActionComposerUtils {
             proposedMetadata: existingMetadata,
             inputData: {
                 function: 'setMetadata',
-                contract: plugin.subdomain,
+                contract: plugin.subdomain ?? plugin.interfaceType,
```

Comment:
If subdomain could be missing here, I would just use `interfaceType`
```suggestion
                contract: plugin.interfaceType,
```

Reply:
removed subdomain here